libblastem.c

```diff
@@ -238,11 +242,43 @@ RETRO_API unsigned retro_get_region(void)
 /* Gets region of memory. */
 RETRO_API void *retro_get_memory_data(unsigned id)
 {
+	if (id == RETRO_MEMORY_SYSTEM_RAM)
```

Comment:
I'm not sure if it figures in to the cheat/rumble support, but cheevos also use `RETRO_MEMORY_SAVE_RAM`. Would be nice to add that while you're in here.

Reply:
thanks for the review. save memory added.